measure_extinction/extdata.py

```diff
@@ -951,6 +951,10 @@ def plot(
             convert from E(lambda-X) using A(X), if necessary
             plot A(lambda)/A(X)
 
+        plot_rel_band : str [default=None]
+            plot relative to this band
+            if None then use the existing relative band (often V)
```

Comment:
Should add that this is only if alax=True (if alax=False, this keyword will be ignored).